common-app-lib/resources/templates/structure.html

```diff
@@ -13,6 +13,7 @@
     <link href="https://cdn.earthdata.nasa.gov/eui/1.1.3/stylesheets/application.css" rel="stylesheet" />
     <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700,300" rel="stylesheet" type="text/css">
     <link href="https://cdn.earthdata.nasa.gov/eui/1.1.3/docs/assets/docs.css" rel="stylesheet" />
```

Comment:
We should definitely talk about this. It's not ideal that you're pointing to this particular stylesheet. I think you're protected from any changes because EUI is versioned, but pointing to a stylesheet for another site isn't the way to go. I realize you said you had some troubles getting things to look right initially when referencing the default EUI stylesheet, but we should revisit that. This would be akin to pointing to MMT's stylesheet, which would never happen.